docs/schema/documentconverter-element-document-converter.md

```diff
@@ -104,7 +105,7 @@ The following sections describe attributes, child elements, and parent elements.
 </colgroup>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="elements-element-document-converter.md">Elements Element (Document Converter)</a></span></p></td>
+<td align="left"><p><span><a href="elements-element-document-converter.md">Elements Element (Document Converter)</a></span></p></td>
```

Comment:
We need to turn this HTML table into a Markdown table.  Ping me if you need help.

Reply:
This one won't convert into a table for some reason.